appserver/appclient/server/pom.xml

```diff
@@ -39,24 +39,28 @@
     only if the new code is made subject to such option by the copyright
     holder.
 
+    Portions Copyright [2019] [Payara Foundation and/or its affiliates]
 -->
 
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
+
     <parent>
-        <groupId>org.glassfish.main</groupId>
+        <groupId>fish.payara</groupId>
         <artifactId>appclient</artifactId>
         <version>5.193-SNAPSHOT</version>
-        <relativePath>../pom.xml</relativePath>
     </parent>
-    <groupId>org.glassfish.main.appclient</groupId>
+
+    <groupId>fish.payara.appclient</groupId>
     <artifactId>server</artifactId>
     <packaging>pom</packaging>
-    <name>GlassFish App Client Server-side Modules</name>  
+
+    <name>Payara App Client - Server-Side Modules</name>  
+
     <modules>
-       <module>connector</module>
-       <module>core</module>
-       <module>core-l10n</module>
-<!--       <module>jws</module> -->
+       <module>appclient-connector</module>
```

Comment:
Considering the naming consistency of other refactored appclient client and server modules. IMHO this module should be renamed to `appclient-server-connector`.

Reply:
I thought about that, but I decided it sounded weird since `server-connector` implies it's for connecting to the server. What do you think?

Reply:
After speaking to @Pandrex247 I've changed my mind and changed it as you said!